### ISSUE TITLE
Permission Problem, Default theme update Problem

### ISSUE THREAD

**Issue by rsm-gh** [rsm-gh/alienware-kbl#1]
By using a non-root account I got the following error:

```
profile added Default
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/share/alienware-kbl/GUI.py", line 700, in ILUMINATE_keyboard
    os.utime(self.configuration.path, None)
PermissionError: [Errno 13] Permission denied
```

**Comment by rsm-gh** [comment]
This was patched as bug #12 . It may be improved since the patch was mostly a hack to ignore setting the `os.utime`.

**Comment by rsm-gh** [comment]
I dont think this problems happens any more under the new version. I'm gonna close it.
